projects/questionset-editor-library/src/lib/components/question/question.component.spec.data.ts

```diff
@@ -151,22 +152,22 @@ export const mockData = {
               {
                 label: "<p>option 1</p>",
                 value: 0,
-                hints: {
-                  en: "test hint 1",
+                hint: {
+                  '70c82bf5-9459-4c43-8897-0e58b7e1da62':'test hint 1'
                 },
               },
               {
                 label: "<p>option 2</p>",
                 value: 1,
-                hints: {
-                  en: "test hint 2",
+                hint: {
```

Comment:
@ramkumar-pacewisdom Please check the above comment

Reply:
updated